packages/react-combobox/src/components/Option/useOption.tsx

```diff
@@ -48,6 +50,12 @@ export const useOption_unstable = (props: OptionProps, ref: React.Ref<HTMLElemen
   const selected = key ? !!selectedOptions.find(option => option.key === key) : false;
   const optionValue = getValueString(value, props.children);
 
+  // check icon
+  let CheckIcon = <CheckmarkFilled />;
```

Comment:
Consider if React.memo would be good to use here.

Reply:
I don't think we memoize icons elsewhere right now, so I think I'll leave as-is for now. It'd be an interesting sync question though :D

Reply:
I know @tomi-msft is looking at icon rendering performance. He may have some guidance here.